blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/ElasticsearchRestHelper.java

```diff
@@ -42,6 +46,10 @@ public class ElasticsearchRestHelper {
             new BasicHeader("Content-Type", "application/json")
     };
 
+    private Histogram availablePoolSize = Metrics.histogram(getClass(), "Available Connections");
+    private Histogram pendingPoolSize = Metrics.histogram(getClass(), "Pending Connections");
+    private Histogram leasedPoolSize = Metrics.histogram(getClass(), "Leased Connections");
```

Comment:
So when you asked about histogram vs gauge, I thought you were talking about the visualization. Aren't these more appropriately gauge metrics, in that they track distinct values that go up and down? A histogram metric is more geared toward collecting values into statistical buckets, I think. I'm not really sure what the difference is in how these are reported, though.

Reply:
Agree. Gauge is correct choice. With few random simulated values confirmed we can view the trend as well as current value (if we need it
<img width="1435" alt="Screen Shot 2022-07-27 at 8 33 20 AM" src="https://user-images.githubusercontent.com/4536063/181260110-a042434e-6e9c-4a1d-91c9-2cb2fbe74e79.png">
)